airflow/executors/base_executor.py

```diff
@@ -117,11 +115,15 @@ def has_task(self, task_instance: TaskInstance) -> bool:
     def sync(self) -> None:
         """
         Sync will get called periodically by the heartbeat method.
+
         Executors should override this to perform gather statuses.
         """
 
     def heartbeat(self) -> None:
-        """Heartbeat sent to trigger new jobs."""
+        """
+        Heartbeat sent to trigger new jobs.
+
```

Comment:
this empty line can be removed.

_(can not do suggestions on subset of changes)_

Reply:
Yep